src/modules/ROOT/pages/changelog/202303/01.json

```diff
@@ -0,0 +1,889 @@
+        "Zoom in to a more appropriate level.",
+        "Before, the map markers were too far off and users had to zoom in all the time to see the exact coordinates."
+      ],
+      "why": [
+        "to prevent a confusion when different markers in multivalue fields were being opened on the map.",
+        "This change aims to improve ux and usability of the geometry field."
+      ],
+      "how": [
+        "Leaflet map instance in compose was updated.",
+        "Zoom level was also updated."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/940/commits/79979c158bd058d0fbcf8e8503c872fbcaf0a0f6"
+      ]
+    },
+    {
+      "what": [
+        "new file configurator option to preview image in modal and option to download.",
+        "Images on the file base are centered now by default."
+      ],
+      "why": [
+        "to center the images on the filebase for better user experience."
+      ],
+      "how": [
+        "by updating the file configurator with the new options."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/937/commits/b3b1bf8f14c26b96396e80aaa8a7c7084f11bc8d",
+        "https://github.com/cortezaproject/corteza/pull/937/commits/24a0a857f4d9f392ae63091481fa07c52d68513b"
+      ]
+    },
+    {
+      "what": [
+        "when no resources are present, the `NaN items` was replaced with `0 items`."
+      ],
+      "why": [
+        "to have consistency, since on every other place when there are no items, we display them as zero."
+      ],
+      "how": [
+        "by updating the resource list configuration."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/999/commits/b7e2d598687dfd92adc2eb15ec9ed6aee7245e84"
+      ]
+    },
+    {
+      "what": [
+        "the configuration for pie and doughnut charts.",
+        "Rotate and metric label fields got hidden since they were not needed."
+      ],
+      "why": [
+        "to have consistency since they were shown but not needed.",
+        "Now they are hidden since we don't need them."
+      ],
+      "how": [
+        "Chart configurations were updated to adjust existing rotate logic for the above-mentioned charts."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/992/commits/d73c93f348cf3f6afcd4b66a3bb8d1c72dab430a"
+      ]
+    },
+    {
+      "what": [
+        "description of handle/namespace slug."
+      ],
+      "why": [
+        "to have consistency across our system and to be more user friendly."
+      ],
+      "how": [
+        "by updating the description of our slugs."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/993/commits/b94b3a7b07f9d83768b0a8dedc49c6bfaf984970"
+      ]
+    },
+    {
+      "what": [
+        "output format when output relative value is selected in date field.",
+        "Now the output format is hidden if relative value is selected."
```

Comment:
Consider merging the two

Reply:
Ohhh the field is hidden? If so, the same comment as the other chart config comment